### ISSUE TITLE
Add menu button to link related resources

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#1310]
We need something like this to accommodate expanding set of resources related to IDC. The importance of this came up again in the recent discussions with the stakeholders.

<img width="376" alt="image" src="https://github.com/ImagingDataCommons/IDC-WebApp/assets/313942/2dafeb18-71bd-4eda-aea3-2826ff5a1d59">

@s-paquette @G-White-ISB can you please comment here on the complexity of adding such a button to the portal?

**Comment by s-paquette** [comment]
MHub https://mhub.ai/
Slicer extension https://github.com/ImagingDataCommons/SlicerIDCBrowser
idc-index https://github.com/ImagingDataCommons/idc-index
zenodo https://zenodo.org/communities/nci-idc/
publications https://learn.canceridc.dev/publications
Documentation https://learn.canceridc.dev/

**Comment by fedorov** [comment]
@s-paquette MHub logo in SVG courtesy @LennyN95 ![logo_green@1x](https://github.com/ImagingDataCommons/IDC-WebApp/assets/313942/c947401b-7864-491f-8535-daf0f50a75b5)

**Comment by LennyN95** [comment]
Thank you @fedorov! Unfortunately, I sent you a flawed export, please use the following (SVG, png).

<img src="https://github.com/ImagingDataCommons/IDC-WebApp/assets/14541510/cb12a41e-14ca-4c96-b346-d504b2b84e4c" width="300" />
<img src="https://github.com/ImagingDataCommons/IDC-WebApp/assets/14541510/37f95c13-fcf9-4b83-8d28-cccc456b751d" width="300" />

**Comment by G-White-ISB** [comment]
Starting work on this now

**Comment by fedorov** [comment]
Let's also include a link to this page that has dashboards related to individual collections: https://learn.canceridc.dev/cookbook/data-studio/more-dashboard-examples

**Comment by fedorov** [comment]
Since @G-White-ISB has been working on this, it is time for @s-paquette to review and take over...

**Comment by s-paquette** [comment]
@fedorov Turns out I was almost done with this. Putting it on dev for you to look at. I grabbed the actual Slicer Extension's icon from the GitHub for that one. Zenodo doesn't appear to have one, though their favicon on their site is the stylized ZE which we could put there.

I went out and found some nicer (free) icons because our current set is a little plain. Let me know if you'd like something different, though; Streamline has a relatively robust library of free options.

**Comment by fedorov** [comment]
@s-paquette here's what I suggest. Note that some of the links were updated as well.

I assume you can update the fontawesome colors to be consistent as a setting?

|![Image](https://github.com/user-attachments/assets/49d07b29-8f96-44c6-bd2f-7be249fc6bfa) | ![Image](https://github.com/user-attachments/assets/8b6c109a-771f-4c8c-9cab-ba39e16a8ab7)|![Image](https://github.com/user-attachments/assets/50f61ab8-3fad-48ca-8a2c-441aa64ef5c5)|
|:---:|:---:|:---:|
|[Documentation](https://learn.canceridc.dev/)|[Tutorials](https://github.com/ImagingDataCommons/IDC-Tutorials)|[Publications](https://learn.canceridc.dev/publications)|
|![Image](https://github.com/user-attachments/assets/f33bf1b6-ee11-4636-8712-67be147f280b)|![Image](https://github.com/user-attachments/assets/f67176c4-3372-4d59-8c18-9112579122b0)|![Image](https://github.com/user-attachments/assets/d9933d35-f04d-4b9e-9b31-ef3ae61d94aa)|
|[Dashboards](https://learn.canceridc.dev/cookbook/data-studio/more-dashboard-examples)|[Forum](https://discourse.canceridc.dev)|[Python API](https://github.com/ImagingDataCommons/idc-index)|
|![Image](https://github.com/user-attachments/assets/cfda2013-4ae8-4f08-aff8-d9a4d0e72d9f)|![Image](https://github.com/user-attachments/assets/bc48c8c5-3195-408f-8ad4-55108398ed63)|![Image](https://github.com/user-attachments/assets/bc29d60e-d185-408d-9998-af2b18bb9b5a)|
|[Zenodo community](https://zenodo.org/communities/nci-idc/about)|[AI Models](https://mhub.ai/)|[3D Slicer extension](https://github.com/ImagingDataCommons/SlicerIDCBrowser)|


https://fontawesome.com/icons/book-open?f=classic&s=solid
https://fontawesome.com/icons/person-chalkboard?f=classic&s=solid
https://fontawesome.com/icons/book?f=classic&s=solid
https://fontawesome.com/icons/discourse?f=classic&s=brands
https://www.slicer.org/assets/img/3D-Slicer-Mark.svg
https://fontawesome.com/icons/chart-bar?f=classic&s=regular

**Comment by s-paquette** [comment]
@fedorov I can set the font to a single solid color which is consistent across them, yes. Zenodo might be a different issue--is that a PNG or an SVG? I don't have that one, just an SVG of the first two lettes. (If it's not a transparent PNG we'll run into the background problem.)

**Comment by s-paquette** [comment]
@fedorov Changes made and now on dev and test--let me know how you find the formatting/layout/sizing. We can swap from a drop shadow to a solid border, for example, add in grid-lines, make the bottom row a little taller, etc. All minor adjustments and easy to make.

**Comment by fedorov** [comment]
@s-paquette this looks great to me!

Unless @ulrikew has objections, I suggest we mark this one as ready to go.

**Comment by ulrikew** [comment]
Looks good to me. Will the items in the menu be labeled the same as in the top across menu, e.g. Forum and User Forum; Documentation and Help?

![Image](https://github.com/user-attachments/assets/a9e9149e-8a47-4584-9e5a-eb00f2ea70aa)

**Comment by fedorov** [comment]
I don't recall the motivation for using "Help" to refer to the documentation, but I have to say it is quite unusual and unconventional. I would prefer to keep "Documentation". I have no concerns changing "Forum" to "User Forum".

**Comment by ulrikew** [comment]
I agree. Documentation sounds better than Help. Forum or User Forum - I don't care if it is consistent.

**Comment by ulrikew** [comment]
@s-paquette Thank you for the explanation. Do we have space for calling it Documentation in the top across menu if we change User Forum to Forum? In the end, I don't really care as long as the naming is consistent and we don't have two links that are labeled differently point to the same place.

**Comment by s-paquette** [comment]
@ulrikew @fedorov Spoke to soon--the grid menu button and the 'Cohorts' link when a user is logged in restrict our size. I'll need to have the homepage roll to the mobile config at about 1345 pixels now (instead of 1245 atm). Is that alright?

We really have a lot of links with long text compared to the other DCs, which is part of the issue we have.

**Comment by ulrikew** [comment]
On a small screen, the top menu disappears. Is that what you mean when you talk about rolling to the mobile config?

Large screen

![Image](https://github.com/user-attachments/assets/7d46e63c-5896-4504-b69e-69a52a4d40a1)

Small screen

![Image](https://github.com/user-attachments/assets/a3e27774-f046-477d-8473-34ef02fc58b0)

**Comment by fedorov** [comment]
@ulrikew that's exactly it. I was going to suggest that perhaps we don't need to have identical labels for the same content. "Help" and "Documentation" can point to the same thing, so that both those who look for Help and Documentation can find these. The advantage of "Help" is that it is shorter, and the top-level menu will not collapse as quickly.

**Comment by ulrikew** [comment]
It is still weird to have the same thing called two different names, but if that is what it takes to make it work, so be it.

**Comment by fedorov** [comment]
I don't have strong feelings either way. I am happy with any solution that you like Ulli.

**Comment by ulrikew** [comment]
@s-paquette  Looks like it is up to you. =:^)
